samples/subsys/power/power_mgr/src/main.c

```diff
@@ -56,5 +57,7 @@ void main(void)
 		}
 	}
 
+	sys_pm_dump_debug_info();
```

Comment:
Do we have any automated way to check if the tests works as expected?

Reply:
as of now no, we might have to add new API or expend the existing one to return the details.